### DISCUSSION TITLE
How to get the coordinates of a vertex?

### DISCUSSION THREAD

**Discussion by NikitaKlimenkoMIT** [mozman/ezdxf#815] [Q&A] [answered]
I have a polyline and I was able to extract vertices. I don't see how I can get the location of a vertex.
Checked this but does not help: https://ezdxf.readthedocs.io/en/stable/dxfentities/polyline.html#vertex 
THank you!

**Comment by mozman** [comment, marked as answer]
```Python
for vertex in polyline.vertices:
     print(f"{str(vertex)} at OCS location {vertex.dxf.location}")
```

**Reply by NikitaKlimenkoMIT** [reply]
thank you!
